extensions/amp-pinterest/0.1/pin-widget.js

```diff
@@ -58,9 +58,9 @@ export class PinWidget {
     const log = el.getAttribute('data-pin-log');
     if (href) {
       if (shouldPop) {
-        openWindowDialog(window, href, '_pinit', POP);
+        openWindowDialog(window, encodeURI(href), '_pinit', POP);
```

Comment:
This won't do it. I can still use a `javascript:alert('PWNED')`, which'll pass right through. We likely want something like `assertAbsoluteHttpOrHttpsUrl`.

Reply:
Taken this out in favor of assertAbsoluteHttpOrHttpsUrl